Day1/que2

```diff
@@ -0,0 +1,66 @@
+2) Create at least 2 feature branches from the main branch (Add multiple files)  - 
```

Comment:
both q2 and q3 files are identical, you can keep only one file with name `que2_que3`